photutils/morphology.py

```diff
@@ -5,14 +5,27 @@
 """
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
+import collections
 import numpy as np
-from astropy.modeling import models
+from astropy.modeling.models import Gaussian1D, Gaussian2D, Const1D, Const2D
 from astropy.modeling.fitting import LevMarLSQFitter
+from astropy.nddata.utils import overlap_slices
 from .segmentation import SegmentProperties
+import warnings
 
 
-__all__ = ['centroid_com', 'gaussian1d_moments', 'centroid_1dg',
-           'centroid_2dg', 'fit_2dgaussian', 'data_properties']
+__all__ = ['GaussianConst1D', 'GaussianConst2D', 'centroid_com',
```

Comment:
I'm going to take them out of the public API.  There's also another issue with the docs for these compound models.  They get base classes called `photutils.morphology.CompoundModel0` and
`photutils.morphology.CompoundModel1`, which do not explicitly exist and therefore do not resolve (giving `Sphinx` warnings):
![untitled](https://cloud.githubusercontent.com/assets/4992897/6137694/0313016c-b149-11e4-8411-c9e6e892bf18.jpg)

@embray:  can the `Sphinx` warnings for compound models be fixed?